File: 프로그래머스/2/12909. 올바른 괄호/올바른 괄호.py
def solution(s):
    answer = True
    check = 0
    for i in range(len(s)):
        if s[i] == '(':
            check += 1
        else:
            if check < 1:
                return False
            else:
                check -= 1
    return True if check == 0 else False

File: 프로그래머스/2/12909. 올바른 괄호/올바른 괄호.js
function solution(s){
    let left = 0;
    for(let i = 0; i < s.length; i++){
        if(s[i] === '('){
            left++;
        }else{
            if(left === 0){
                return false;
            }
            left--;
        }
    }
    return left === 0;
}